src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -727,8 +728,16 @@ sqlWhereIsNULLE :: (MonadState v m, SqlWhere v, DBExtraException e, FromSQL a)
 sqlWhereIsNULLE exc col = sqlWhereEV (exc, col) $ col <+> "IS NULL"
 
 sqlWhereAny :: (MonadState v m, SqlWhere v) => [State SqlAll ()] -> m ()
-sqlWhereAny [] = sqlWhere "FALSE"
-sqlWhereAny l = sqlWhere $ "(" <+> smintercalate "OR" (map (parenthesize . toSQLCommand . flip execState (SqlAll [])) l) <+> ")"
+sqlWhereAny l = do
+  let sql = sqlForWhereAny l
+  sqlWhereE (DBBaseLineConditionIsFalse sql) sql
+
+sqlWhereAnyE :: (DBExtraException e, MonadState v m, SqlWhere v) => e -> [State SqlAll ()] -> m ()
+sqlWhereAnyE e = sqlWhereE e . sqlForWhereAny
+
+sqlForWhereAny :: [State SqlAll ()] -> SQL
```

Comment:
> `[State SqlAll ()]`

From looking at the code, it seems that it would make sense to add a `SqlAny` newtype as well, though that's a bigger refactoring (and therefore not required for this PR to be merged).